instrumentation/graphql_batch/Appraisals

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+appraise 'graphql-batch-0.5.1' do
+  gem 'graphql-batch', '~> 0.5.1'
+end
```

Comment:
Should you add an appraisal for "latest", too?

```
appraise 'graphql-batch-latest' do
  gem 'graphql-batch'
end
```

Or even "edge":

```
appraise 'graphql-batch-edge' do
  gem 'graphql-batch', git: 'https://github.com/shopify/graphql-batch'
end
```

Reply:
Yep, I'll update this. We'd only support from version 0.5.3 since that's when the `around_perform` wrapper was added.